jwcrypto/jwa.py

```diff
@@ -599,10 +597,7 @@ class _Pbes2HsAesKw(_RawKeyMgmt):
 
     def __init__(self):
         self.backend = default_backend()
-
-    @property
-    def min_key_size(self):
-        return self.keysize * 8
+        self.aeskwmap = {128: _A128KW, 192: _A192KW, 256: _A256KW}
```

Comment:
Since this class is defined after the AES classes, you can move the dict into the class body. I'd prefix it was _ to make clear that it is an internal API.